lib/src/components/_stacks-tables.less

```diff
@@ -115,22 +133,61 @@
         }
     }
 
-    //  $$  Bordered
+    //  $$  Expanded Padding
     //  ------------------------------------------------------------------------
-    &.s-table__bordered {
+    &.s-table__expanded {
         th,
         td {
-            border: 1px solid @brc-lighter;
+            padding: @su12;
         }
+    }
 
+    //  $$  Bordered
+    //  ------------------------------------------------------------------------
+    &.s-table__borderless {
+        th,
+        td {
+            border-top: none;
+            border-right: none;
+        }
         thead th {
-            border-bottom-color: @brc-darker;
+            border-bottom: none;
+        }
+        tbody tr:last-child th,
+        tfoot th,
+        tr:last-child td {
+            border-bottom: none;
         }
-
         tbody + tbody {
-            border-top: 2px solid @brc-darker;
+            border-top: none;
         }
     }
+    &.s-table__borderless-x {
```

Comment:
I would change this name to `.s-table__bx0`. It brings it more line in with how we do directional things like this with padding and margins.